staging_service/AutoDetectUtils.py

```diff
@@ -6,7 +6,6 @@
 
 
 class AutoDetectUtils:
-    _FILE_EXTENSION_MAPPINGS = None  # expects to be set by config
```

Comment:
This is set but never actually used, so *bonk*

Reply:
It's being used here https://github.com/kbase/staging_service/blob/develop/staging_service/app.py#L493-L497

Reply:
I switched those to just use `FILE_EXTENSION_MAPPINGS`